source/_docs/installation/docker.markdown

```diff
@@ -8,6 +8,8 @@ redirect_from: /getting-started/installation-docker/
 
 These below instructions are for an installation of Home Assistant Core running in your own Docker environment, which you manage yourself.
 
+Note that docker comand line option `--net=host` or the compose file equivalent `network_mode: host` must be used to put it on the host's network, otherwise mDNS and UPnP will not work.
+
```

Comment:
Let's generalize this a bit more, as it is not limited to mDNS & UPnP.

Reply:
So simply state "Note that docker command line option `--net=host` or the compose file equivalent `network_mode: host` must be used to put it Home Assistant on the host's network." or maybe "Note that docker command line option `--net=host` or the compose file equivalent `network_mode: host` must be used to put Home Assistant on the host's network, otherwise certain functionality will break."?

Reply:
The last line sets a "why" at least, so let's do that.